src/OmniSharp.Roslyn.CSharp/Helpers/SourceTextExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿#nullable enable
+
+using Microsoft.CodeAnalysis.Text;
+using OmniSharp.Models;
+
+namespace OmniSharp.Roslyn.CSharp.Helpers
+{
+    internal static class SourceTextExtensions
+    {
+        public static int GetTextPosition(this SourceText sourceText, Request request)
+            => sourceText.Lines.GetPosition(new LinePosition(request.Line, request.Column));
```

Comment:
I think we have a line like `sourceText.Lines.GetPosition(new LinePosition(request.Line, request.Column));` at least 10 times in different places, since you added this nice extension method, could you replace those usages too? 🙂